Source/WebKit/WebProcess/Network/webrtc/LibWebRTCSocketFactory.cpp

```diff
@@ -102,7 +102,7 @@ rtc::AsyncPacketSocket* LibWebRTCSocketFactory::createClientTcpSocket(WebCore::S
         });
     }
 
-    return socket.release();
+    return socket.leakRef();
```

Comment:
Personally, I doubt this patch is correct (cc @youennf). Calling leakRef() here and passing it to libwebrtc as a `rtc::AsyncPacketSocket* ` likely means libwebrtc will call `delete` on the object when they don't need it anymore, completely ignoring our ref-counting?

Reply:
I see. Would the correct thing be to use `releaseNonNull`?

Reply:
We're implementing an interface required by libwebrtc, which expects a `rtc::AsyncPacketSocket*` so you cannot return a smart pointer. They expect a raw pointer to the base class. libwebrtc also doesn't know that the object is ref-counted so I'm pretty sure they'll just end up calling `delete` when done.

I think this object (and the other one above) simply CANNOT be ref-counted since we don't control the way the object is used / destroyed.

Reply:
Ok that does make sense. Hopefully this can be fixed by making LibWebRTCSocket able to be a CheckedPtr. We'll see if the bots are happy with this.